src/apps/api/views/competitions.py

```diff
@@ -337,6 +337,27 @@ def create_dump(self, request, pk=None):
         serializer = CompetitionCreationTaskStatusSerializer({"status": "Success. Competition dump is being created."})
         return Response(serializer.data, status=201)
 
+    def run_new_task_submissions(self, phase, tasks):
+        tasks_id = set([task.id for task in tasks])
+        submissions = phase.submissions.prefetch_related("children").all()
+
+        for submission in filter(lambda s: s.has_children, submissions):
+            child_tasks = set([sub.task for sub in submission.children.all()])
+            missing_tasks = tasks_id - set([task.id for task in child_tasks])
+            for task in filter(lambda t: t.id in missing_tasks, tasks):
+                sub = Submission(
+                    owner=submission.owner,
+                    phase=submission.phase,
+                    parent=submission,
+                    task=task,
+                    appear_on_leaderboards=submission.appear_on_leaderboards,
+                    leaderboard=submission.children.first().leaderboard,
+                    data=submission.data,
+                )
+                sub.save(ignore_submission_limit=True)
+                sub.refresh_from_db()
```

Comment:
comment on why we need this here?

Reply:
I'm not seeing a reason for the `sub.refresh_from_db()` that happens immediately after `sub.save(...)`.